Symfony/CS/FixerInterface.php

```diff
@@ -75,6 +75,8 @@ public function getPriority();
     /**
      * Returns true if the file is supported by this fixer.
      *
+     * @param \SplFileInfo $file A \SplFileInfo instance
```

Comment:
How is this different from:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/Symfony/CS/FixerInterface.php#L31
?
(or from the hinting to the Tokens below that?)